google-services.json

```diff
@@ -0,0 +1,40 @@
+{
+  "project_info": {
+    "project_number": "810662302565",
+    "firebase_url": "https://templetag-8fe75.firebaseio.com",
+    "project_id": "templetag-8fe75",
+    "storage_bucket": "templetag-8fe75.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:810662302565:android:c8071029e9c6c6b307478c",
+        "android_client_info": {
+          "package_name": "edu.temple.templetag"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "810662302565-es5ritsgprpo6lhsbu2cl3oai1ajv5rq.apps.googleusercontent.com",
+          "client_type": 3
+        }
+      ],
+      "api_key": [
```

Comment:
Firebase API key is not a security risk: https://stackoverflow.com/questions/37482366/is-it-safe-to-expose-firebase-apikey-to-the-public